src/main/java/com/systemmeltdown/robotlib/subsystems/SingleSpeedTalonDriveSub.java

```diff
@@ -0,0 +1,194 @@
+package com.systemmeltdown.robotlib.subsystems;
+
+import java.util.ArrayList;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.StatusFrame;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.systemmeltdown.robotlib.util.PIDValues;
+import com.systemmeltdown.robotlib.util.Utility;
+
+public class SingleSpeedTalonDriveSub extends SkidSteerDriveSubBase {
+    WPI_TalonSRX m_rightMaster;
+    WPI_TalonSRX m_leftMaster;
+    WPI_TalonSRX[] m_rightSlaves;
+    WPI_TalonSRX[] m_leftSlaves;
+    int m_timeout;
+    int m_talonPidPrimary;
+    double m_driveMotorDeadband;
+    double m_driveRampSeconds;
+
+    int m_talonSlotVelocity;
+    int m_talonSlotDistance;
+    int m_talonSlotTurning;
+    PIDValues m_pidDrivePos;
+
+    PIDValues m_pidDriveSpeed;
+
+    public SingleSpeedTalonDriveSub(int rightMasterID, int leftMasterID, int[] rightSlaveIDS, int[] leftSlaveIDS,
+            int timeout, int talonPidPrimary, double driveMotorDeadband, double driveRampSeconds, int talonSlotVelocity,
```

Comment:
I wonder if we should make a config object to pass in here. I think it would simplify this instantiation.